supabase/migrations/20240130152000_create_model.sql

```diff
@@ -35,5 +35,5 @@ CREATE TABLE IF NOT EXISTS games (
     second_player_id INT REFERENCES users(id) ON DELETE CASCADE,
     creation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     last_update_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    game_state VARCHAR(255) NOT NULL
-);
\ No newline at end of file
+    game_state JSONB NOT NULL
```

Comment:
This still needs to be it's own migration I'm afraid :grimacing: or, at least, we should warn the client that they will need to drop their DB and recreate it from scratch to test this version when it is merged.